packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -75,8 +75,7 @@ startTelemetry(telemetryConfig);
 
 That's it for now! If you've decided to use Azure App Insights, we have designed useful prebuilt queries that utilize the generated telemetry. You can try out these queries after running your application with above code for a little duration.
 
-You can find these queries [here]().
-// TODO: rohandubal:Fix me
+You can find these queries [here](#telemetry_visualization).
```

Comment:
Nit, I was told once that for screen readers it is useful to have more descriptive text in links.
 ```suggestion
You can find these queries in [the "Interpreting telemetry data" section](#telemetry_visualization) below.
```